src/layout/Header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import { Container } from '../../components/ui/Container';
+import { HeaderStyled, BackButton, HeaderTitle } from './Header.styled';
+import { useLocation } from 'react-router-dom';
+
+const Header: FC = () => {
+  const nav = useLocation();
+  const isEditPage = nav.pathname.includes('edit');
+
+  return (
+    <Container>
+      <HeaderStyled>
+        {isEditPage ? <BackButton to="/">Back</BackButton> : <div></div>}
```

Comment:
returning an empty div is a bad practice, please replace it with logical AND.

This component should be stateless and should be placed in the components folder.
It might accept the following props:
action (if it passed - show BackButton)
children (which will be HeaderTitle)